README.md

```diff
@@ -6,6 +6,7 @@ This project implements a generic and customizable text tokenization based on th
 
 * Fast and generic text tokenization with minimal dependencies
 * Support for BPE or SentencePiece models
+* Efficient learner mode for learning subword models
```

Comment:
Just for reference: did you measure the speedup compared to the Python implementation?

Reply:
On joint en/nl corpus (620k sentences, 13M tokens (aggressive tokenization)), I obtain 41s for python subword-nmt/learn.py (default params), vs. 20s for subword_learn. Interestingly with same untokenized corpus, and using on the fly tokenization, total time is 29s.